docs/source/refs/issues.rst

```diff
@@ -1,6 +1,40 @@
 Known issues
 ============
 
+.. attention::
+
+    Please also refer to the `Omniverse Isaac Sim documentation`_ for known issues and workarounds.
+
+Stale values after resetting the environment
+--------------------------------------------
+
+When resetting the environment, some of the data fields of assets and sensors are not updated.
+These include the poses of links in a kinematic chain, the camera images, the contact sensor readings,
+and the lidar point clouds. This is a known issue and which has to do with the way the PhysX and
+rendering engines work in Omniverse.
+
+Many physics engines do a simulation step as a two-level call: ``forward()`` and ``simulate()``,
+where the kinematic and dynamic states are updated, respectively. Unfortunately, PhysX has only a
+single ``step()`` call where the two operations are combined. Due to computations through GPU
+kernels, it is not so straightforward for them to split these operations. Thus, at the moment,
+it is not possible to set the root and/or joint states and do a forward call to update the
+kinematic states of links. This affects both initialization as well as episodic resets.
+
+Similarly for RTX rendering related sensors, the sensor data is not updated immediately after
```

Comment:
NIT: Can we give some examples of RTX rendering related sensors here? Is this just cameras and raycasters? I'm not familiar with this and other users may not be either
```suggestion
Similarly for RTX rendering related sensors, the sensor data is not updated immediately after
```